mappings/ae82474613be1ebc28b7ed8ebf60483768f036d5ce6caee53076fe324c6f76656c61636547656e65736973416461.json

```diff
@@ -0,0 +1,54 @@
+{
+    "subject": "ae82474613be1ebc28b7ed8ebf60483768f036d5ce6caee53076fe324c6f76656c61636547656e65736973416461",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://yifupedersen.com/Genesis",
+        "signatures": [
+            {
+                "signature": "c93c1815f16a3b2a2de7607d3ac54f6e556374e0b38c1a4dfdcc1f783daa9ef67203485df13c917d90d5aadc863758bc4e8c32715c44a181227644778788d00e",
+                "publicKey": "56e52537b97913e9b3aa4a115118fa72641f67da9968921a3d44a01988792691"
+            }
+        ]
+    },
+    "name": {
+        "sequenceNumber": 0,
+        "value": "Ada",
```

Comment:
@LeadTimeNull - naming a token "Ada", "Cardano" or other will likely trigger a more involved legal review - 
(see [here](https://github.com/cardano-foundation/cardano-token-registry/blob/master/Registry_Terms_of_Use.md#permitted-use-of-the-cardano-registry))
I propose you change the ambiguous terms used, to facilitate the merge of the mapping

Reply:
I see. The token in question is named LovelaceGenesisAda, as its hexdump will verify, and it's 'name' field value is it's shorthand name or nickname if you would.

It is a 1/1 token that represents a physical product (see for reference: https://pool.pm/ae82474613be1ebc28b7ed8ebf60483768f036d5ce6caee53076fe32.LovelaceGenesisAda) 

The use of the Cardano logo and likeness has been requested and approved back in March in a correspondence with Amanda Rasch from Cardano Foundation.

I would like to proceed with the legal review of this.

Reply:
Note - this PR was flagged for Legal review, but understand that it will not be merged as-is. 
In the current state of things we are requesting a single Mapping be submitted per PR due to the manual processing/review taking place. 

=> Accordingly, please kindly split this PR into separate PRs (with a single mapping file for each) so they can be separately reviewed and processed.

Reply:
Will do ✅

@crptmppt Would you like me to leave this PR for convenience for the Legal review, and commit the individual PRs in separate branches? 

This PR can be closed / archived as suits you the best to maintain track for the legal review.

Reply:
It will be simpler closing this one and with the individual PRs open - thank you for your understanding

Reply:
understood.